charts/trusted-artifact-signer/templates/segment-backup-job-sa.yaml

```diff
@@ -4,4 +4,4 @@ metadata:
   name: segment-backup-job
```

Comment:
Can we also parameterize this with the `name` defined for `configs.segment_backup_job.name`?

Reply:
ack, also added this for the clusterRole and clusterRoleBinding